osbs/api.py

```diff
@@ -1240,21 +1231,25 @@ def get_compression_extension(self):
 
         :returns: str including leading dot, or else None if no compression
         """
-
-        build_request = BuildRequest(build_json_store=self.os_conf.get_build_json_store())
-        inner = build_request.inner_template
-        postbuild_plugins = inner.get('postbuild_plugins', [])
-        for plugin in postbuild_plugins:
+        build_request = BuildRequestV2(build_json_store=self.os_conf.get_build_json_store())
+        build_request.set_params(git_uri="/", git_ref="HEAD", git_branch="master", user="user",
+                                 build_image=self.build_conf.get_build_image(),
+                                 build_imagestream=self.build_conf.get_build_imagestream(),
+                                 build_from=self.build_conf.get_build_from(),
+                                 base_image="base_image", name_label='name', build_type='worker',
+                                 registry_uris=self.build_conf.get_registry_uris())
+
+        plugins_config = PluginsConfiguration(build_request.user_params)
+        build_json = json.loads(plugins_config.render())
```

Comment:
I'm not a huge fan of getting the JSON from `plugins_config` and then converting it back to a dict immediately, though I wasn't able to find a better way. Technically, you could do this
```python
plugins_config.render()
build_json = plugins_config.pt.template
```
but that relies entirely on the side effects of `plugins_config.render()` and I think I like that even less then the ~~Hobbit:~~ JSON: There and Back Again. Probably better to keep it as it is for now, up to you.

Reply:
converting dictionaries back and forth is the least sketchy element of this particular function.  I'll leave it as is.

Reply:
actually, I'll just kill the function entirely.  Even better solution!